src/components/pages/Memos/MemosForm.js

```diff
@@ -181,8 +182,4 @@ const MemosForm = ({ displayModal, setDisplayModal, userProfile }) => {
   );
 };
 
-const mapStateToProps = state => {
-  return { userProfile: state.user.userProfile };
-};
-
-export default connect(mapStateToProps)(MemosForm);
+export default MemosForm;
```

Comment:
Great job going through the props on MemosForm to just export the component as default!

Reply:
Yeah, after taking out what was no longer necessary it became clear that the component didn't need any global state props